js/src/components/paid-ads/audience-section.js

```diff
@@ -2,25 +2,71 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { SelectControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
+import useCountryKeyNameMap from '.~/hooks/useCountryKeyNameMap';
 import Section from '.~/wcdl/section';
 import AudienceCountrySelect from '.~/components/audience-country-select';
 
+function toCountryOptions( countryCodes, countryNameMap ) {
+	return countryCodes.map( ( code ) => ( {
+		label: countryNameMap[ code ],
+		value: code,
+	} ) );
+}
+
+/**
+ * Renders <Section> and <Section.Card> UI with country(s) selector.
+ *
+ * @param {Object} props React props.
+ * @param {Object} props.formProps Form props forwarded from `Form` component.
+ * @param {boolean} [props.multiple=true] Whether the selector is multi-selected.
+ * @param {boolean} [props.disabled=false] Whether the selector is disabled.
+ * @param {JSX.Element} [props.countrySelectHelperText] Helper text to be displayed under the selector.
+ */
 const AudienceSection = ( props ) => {
 	const {
 		formProps: { getInputProps },
+		multiple = true,
 		disabled = false,
 		countrySelectHelperText,
 	} = props;
 
+	const countryNameMap = useCountryKeyNameMap();
 	const inputProps = getInputProps( 'country' );
+
+	/**
+	 * Hack to prevent tags being removed even when disabled.
+	 * This could be removed after replacing <AudienceCountrySelect> with <TreeSelectControl>.
+	 */
 	if ( disabled ) {
 		inputProps.onChange = undefined;
 	}
 
+	const selector = multiple ? (
+		<AudienceCountrySelect
+			label={ __( 'Select countries', 'google-listings-and-ads' ) }
+			helperText={ countrySelectHelperText }
+			disabled={ disabled }
+			multiple
+			isSearchable={ ! disabled }
+			{ ...inputProps }
+		/>
+	) : (
+		<SelectControl
+			label={ __( 'Select one country', 'google-listings-and-ads' ) }
+			help={ countrySelectHelperText }
```

Comment:
💅  ❔  Related to `countrySelectHelperText`, same thing, we don't need it as props, we can have the text inline here, like: 

```suggestion
			helperText={ __(
				'Once a campaign has been created, you cannot change the target country.',
				'google-listings-and-ads'
		  ) }
```

Reply:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/1268#discussion_r813547502

Reply:
Similar to https://github.com/woocommerce/google-listings-and-ads/pull/1268#discussion_r813954523 😆 

```suggestion
			helperText={
				disabled &&
				__(
					'Once a campaign has been created, you cannot change the target country.',
					'google-listings-and-ads'
				)
			}
```